server/compaction-coordinator/src/main/java/org/apache/accumulo/coordinator/CompactionCoordinator.java

```diff
@@ -187,8 +188,9 @@ protected CompactionCoordinator(ServerOpts opts, String[] args, AccumuloConfigur
     printStartupMsg();
     startCompactionCleaner();
     startRunningCleaner();
-    compactorCounts = Caffeine.newBuilder().expireAfterWrite(30, TimeUnit.SECONDS)
+    compactorCounts = Caffeine.newBuilder().expireAfterWrite(120, TimeUnit.SECONDS)
```

Comment:
Easy math, but this seems easier to read.

```suggestion
    compactorCounts = Caffeine.newBuilder().expireAfterWrite(2, TimeUnit.MINUTES)
```